bin/fsl_imglob.py

```diff
@@ -0,0 +1,138 @@
+#!/usr/bin/env python
+#   imglob - expand list of image filenames
+#   Stephen Smith, Mark Jenkinson and Matthew Webster FMRIB Image Analysis Group
+#   Copyright (C) 2009 University of Oxford
```

Comment:
Good point, I forgot to mention. This script was only python 2. But it is required by the eddy correct pipeline (and thus, it is necessary in the circleci tests).

What should we do with it? As I said, the only changes are for python 3 compatibility.